src/custom/components/SearchModal/Manage/ManageMod.tsx

```diff
@@ -0,0 +1,98 @@
+import { Trans } from '@lingui/macro'
+import React, { useState } from 'react'
+import { PaddedColumn, Separator } from '@src/components/SearchModal/styleds'
+import { RowBetween } from 'components/Row'
+import { ArrowLeft } from 'react-feather'
+import { Text } from 'rebass'
+import { CloseIcon } from 'theme'
+import styled from 'styled-components/macro'
+// import { Token } from '@uniswap/sdk-core'
+import { ManageLists } from 'components/SearchModal/ManageLists'
+import ManageTokens from 'components/SearchModal/ManageTokens'
+// import { TokenList } from '@uniswap/token-lists'
+import { CurrencyModalView } from '@src/components/SearchModal/CurrencySearchModal'
+import { ManageProps } from './index' // mod
+
+const Wrapper = styled.div`
+  width: 100%;
+  position: relative;
+  padding-bottom: 80px;
+`
+
+// const ToggleWrapper = styled(RowBetween)`
+//   background-color: ${({ theme }) => theme.bg3};
+//   border-radius: 12px;
+//   padding: 6px;
+// `
+
+// const ToggleOption = styled.div<{ active?: boolean }>`
+//   width: 48%;
+//   padding: 10px;
+//   display: flex;
+//   align-items: center;
+//   justify-content: center;
+//   border-radius: 12px;
+//   font-weight: 600;
+//   background-color: ${({ theme, active }) => (active ? theme.bg1 : theme.bg3)};
+//   color: ${({ theme, active }) => (active ? theme.text1 : theme.text2)};
+//   user-select: none;
+
+//   :hover {
+//     cursor: pointer;
+//     opacity: 0.7;
+//   }
+// `
+
+export default function Manage({
+  onDismiss,
+  setModalView,
+  setImportList,
+  setImportToken,
+  setListUrl,
+  ToggleOption, // mod
+  ToggleWrapper, // mod
+}: ManageProps) {
+  // mod
```

Comment:
Can remove this comment, the ones above are good as they are inline comments and clearly showing what's added. This one here is obv cause you commented it out

Reply:
Updated.